core/src/main/java/com/linecorp/armeria/client/retry/RetryingClient.java

```diff
@@ -32,6 +32,7 @@
  */
 public abstract class RetryingClient<I extends Request, O extends Response>
         extends SimpleDecoratingClient<I, O> {
+    private static final int DEFAULT_MAX_ATTEMPTS = 5;
```

Comment:
Oops~ I misunderstood about system property. I will add the parameter in the construction method and make it to be able to set from java system property.